internal/openapi3/openapi3.go

```diff
@@ -0,0 +1,225 @@
+package openapi3
```

Comment:
We should include a link to the license of the project we based this on.
https://github.com/getkin/kin-openapi/blob/master/LICENSE

Reply:
Ya, possibly. In this case I suspect the license doesn't apply. This is only data structs which are based on the openapi spec. And even those structs are heavily modified.

I'm thinking of maybe removing more of the unused fields, removing the `yaml` struct tags, and changing the comments further, at which point I think it's no longer even close to the original source.